ex2/ex2.c

```diff
@@ -9,6 +9,25 @@
 int main(void)
 {
     // Your code here 
-    
+    FILE *fp;
+    fp = fopen("text.txt", "w");
+    int rc = fork();
+
+    if (rc < 0) {
+        fprintf(stderr, "fork failed\n");
+        exit(1);
+    } else if (rc == 0) {
+        printf("child here\n");
+        char *child_str = "me child\n";
+        fwrite(child_str, sizeof(char), strlen(child_str), fp);
+        // fprintf(fp, child_str);
+    } else {
+        printf("parent here\n");
+        char *parent_str = "me parent\n";
+        fwrite(parent_str, sizeof(char), strlen(parent_str), fp);
```

Comment:
I think in order to use strlen correctly here you have to #include string.h up top. I am getting implicit declaration errors and I think it's because the compiler is not sure what to do with strlen.